core/src/crds_value.rs

```diff
@@ -83,7 +83,7 @@ impl Sanitize for CrdsData {
             CrdsData::ContactInfo(val) => val.sanitize(),
             CrdsData::Vote(ix, val) => {
                 if *ix >= MAX_VOTES {
-                    return Err(SanitizeError::Failed);
+                    return Err(SanitizeError::ValueOutOfRange);
```

Comment:
Vote has an index and a value, and it's the index that's out of range, not the value.  Also, how about "bounds" instead of "range"?  So `IndexOutOfBounds`?